fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -927,7 +946,18 @@ private void executeAnalyze(ConnectContext statsConnectCtx, AnalyzeStmt analyzeS
                                 Database db, Table table) {
         StatisticExecutor statisticExecutor = new StatisticExecutor();
         if (analyzeStmt.isExternal()) {
-            // deal later
+            StatsConstants.AnalyzeType analyzeType = analyzeStmt.isSample() ? StatsConstants.AnalyzeType.SAMPLE :
```

Comment:
Only support FULL, why could set SAMPLE

Reply:
It's impossible, we have checked type in analyze phase.